06_create_cluster.sh

```diff
@@ -26,4 +26,11 @@ done
 oc patch configs.imageregistry.operator.openshift.io \
     cluster --type merge --patch '{"spec":{"storage":{"emptyDir":{}},"managementState":"Managed"}}'
 
+if [[ ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then        
+    # Configure tools image registry and cluster samples operator 
+    # when local image stream is enabled. This is basically to run CI tests
+    # depend on tools image.
+    configure_samples_operator
```

Comment:
I guess the point here is to add the trusted certificate regardless of any other possible additional configuration required for some specific operator (in fact, it's something needed also for sig-builds tests, for example), so it could be something like:
```
add_local_certificate_as_trusted
configure_samples_operator
```
We could not exclude, in future, to have the need to provide some additional configuration for some other operator, and still require the ca trust enabled globally.

Reply:
Good point and agree. I'll change that way.

Reply:
I'd like to note that `configure_sample_operator` is the importer of the `add_local_certificate_as_trusted`. In connected configuration, it seems like they are not dependent but actually it is not that way. It is working because cluster samples operator is already in managed state.